File: VB/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!"

			Return View(New MyModel() With {.Id = 1, .Name = "Test", .Age = 17})
		End Function

		<HttpPost> _
		Public Function PostData(<ModelBinder(GetType(DevExpress.Web.Mvc.DevExpressEditorsBinder))> ByVal model As MyModel) As ActionResult
			If (Not ModelState.IsValid) Then
				Return PartialView("Editors", model)
			End If

			Return PartialView("Success")
		End Function
	End Class
End Namespace

File: CS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";

            return View(new MyModel() { Id = 1, Name = "Test", Age = 17 });
        }

        [HttpPost]
        public ActionResult PostData([ModelBinder(typeof(DevExpress.Web.Mvc.DevExpressEditorsBinder))] MyModel model) {
            if (!ModelState.IsValid)
                return PartialView("Editors", model);

            return PartialView("Success");
        }
    }
}
